test/unit/pr_checker.test.js

```diff
@@ -193,6 +193,42 @@ describe('PRChecker', () => {
       assert(!status);
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
+
+    it('should skip wait check for Code & Learn PR', () => {
+      const logger = new TestLogger();
+
+      const expectedLogs = {
+        warn: [],
+        info: [['This is a Code & Learn PR, it doesn\'t have a time limit.']],
```

Comment:
@Tiriel May be simpify this to `This is Code & Learn PR.`

Reply:
Well, it wouldn't be very explicit as to what it's supposed to replace. In that case, we might just as well remove completely the phrase.
But since it's supposed to skip the time-before-landing limit, I thought it would be better to state clearly what we were skipping, or else silently skip it.

In terms of ux, I'd say more information is better than less :)

Reply:
Sold! I like your last ;)